File: scripts/py-compile-suite/inputs/07_matrix_ops.m
% Matrix operations with transpose
A = [1 2 3; 4 5 6; 7 8 9];
B = A';
C = A * B;
D = A + A;
E = A - eye(3);
F = A \ ones(3, 1);
trace_val = trace(A);
det_val = det(A);

File: scripts/py-compile-suite/outputs/07_matrix_ops.py
import numpy as np

# Matrix operations with transpose
A = np.array([[1, 2, 3], [4, 5, 6], [7, 8, 9]])
B = A.T
C = A @ B
D = A + A
E = A - np.eye(3)
F = np.linalg.solve(A, np.ones(3))
trace_val = np.trace(A)
det_val = np.linalg.det(A)
